src/modules/edc-demo/components/transfer-history/transfer-history-viewer.component.ts

```diff
@@ -59,25 +63,26 @@ export class TransferHistoryViewerComponent implements OnInit {
     });
   }
 
-  // showStorageExplorerLink(transferProcess: TransferProcessDto) {
-  //     return transferProcess.dataDestination?.properties?.type === 'AzureStorage' && transferProcess.state === 'COMPLETED';
-  // }
-  //
-  // showDeprovisionButton(transferProcess: TransferProcessDto) {
-  //     return ['COMPLETED', 'PROVISIONED', 'REQUESTED', 'REQUESTED_ACK', 'IN_PROGRESS', 'STREAMING'].includes(transferProcess.state);
-  // }
-
   loadTransferProcesses() {
-    this.transferProcesses$ = this.transferProcessService
+    this.transferProcessService
       .getAllTransferProcesses()
       .pipe(
-        map((transferProcesses) =>
-          transferProcesses.sort(function (a, b) {
-            return (
-              b.createdTimestamp?.valueOf()! - a.createdTimestamp?.valueOf()!
-            );
+        map(
+          (transferProcesses): TransferProcessesList => ({
+            transferProcesses: transferProcesses.sort(function (a, b) {
+              return (
+                b.createdTimestamp?.valueOf()! - a.createdTimestamp?.valueOf()!
+              );
+            }),
           }),
         ),
+        Fetched.wrap({
+          failureMessage: 'FAILED FETCHING TRANSFERRED PROCESS DETAILS',
```

Comment:
capitalization

"Failed fetching transfer history." there are no details, we are fetching the list